src/friendly-iframe-embed.js

```diff
@@ -550,6 +550,21 @@ export class FriendlyIframeEmbed {
     );
   }
 
+  /**
+   * Removes all resources belonging to the FIE window.
+   * @private
+   */
+  removeResources_() {
```

Comment:
IMHO, a much better option is when a FIE ampdoc gets its own Resources instance that can be completely thrown out.

Reply:
Agree. This PR still brings value though, as once this `childwindow` concept is moved out of resources service, we don't need to worry about providing a dummy impl for inabox runtime.

Reply:
@dvoytenko this is a pure dependency clean up, I think we should get it in. No?